mobile/src/main/java/org/openhab/habdroid/model/Widget.kt

```diff
@@ -217,6 +228,7 @@ fun Node.collectWidgets(parent: Widget?): List<Widget> {
     var step = 1f
     var refresh = 0
     var height = 0
+    var inputHint = "text"
```

Comment:
No need to updated `Node.collectWidgets()`. `Node` refers to xml nodes, so this function is used to parse the OH1 api.

Reply:
Removed